crates/core/src/host/v8/mod.rs

```
@@ -295,6 +295,13 @@ impl JsInstance {
 
         response
     }
+
+    pub async fn call_procedure(
+        &mut self,
+        _params: CallProcedureParams,
+    ) -> Result<super::ProcedureCallResult, super::ProcedureCallError> {
+        todo!("JS/TS module procedure support")
```

Can you return an unimplemented error in a non-panicking way?